File: 1. Two Sum/Solution.java
class Solution {
    public static int[] twoSum(int[] nums, int target) {
        for (int i = 0; i < nums.length; i++) {
            for (int j = i + 1; j < nums.length; j++) {
                if (nums[i] + nums[j] == target) {
                    return new int[] {i, j};
                }
            }
        }
        return new int[] {-1, -1};
    }

    public static void main(String[] args) {
        int[] nums = {3, 2, 4, 9, 8, 105, 42, 57, 12};
        int target = 6;

        int[] result = twoSum(nums, target);
        
        System.out.println("[" + result[0] + ", " + result[1] + "]");
    }
}

File: 1. Two Sum/Solution.py
class Solution:
    def twoSum(self, nums: list[int], target: int) -> list[int]:
        for i in range(0, len(nums)):
            for j in range(i + 1, len(nums)):
                sum = nums[i] + nums[j]
                if sum == target:
                    return [i, j]
        

nums = [3, 2, 4, 9, 8, 105, 42, 57, 12]
target = 114

my_solution = Solution
res = my_solution.twoSum(my_solution, nums, target)

print(res)